src/main/java/com/olgaboiar/mint/handlers/HEADHandler.java

```diff
@@ -0,0 +1,17 @@
+package com.olgaboiar.mint.handlers;
+
+import com.olgaboiar.mint.Request;
+import com.olgaboiar.mint.Response;
+import com.olgaboiar.mint.ResponseGenerator;
+
+import java.io.IOException;
+
+import static com.olgaboiar.mint.Constants.STATUS_CODE_200;
+
+public class HEADHandler implements IHandler {
```

Comment:
Java class name convention follows camelcase, which would be `HeadHandler`. I am not totally against `HEADHandler` since I know where you are coming from, but it could be jarring for Java programmers.